lib/rspec/rails/configuration.rb

```diff
@@ -48,6 +48,15 @@ def self.initialize_configuration(config)
       config.include RSpec::Rails::ViewExampleGroup,       :type => :view
       config.include RSpec::Rails::FeatureExampleGroup,    :type => :feature
 
+      config.before :suite do
+        return unless defined?(ActiveRecord) && defined?(::RSpec::Mocks)
```

Comment:
Fair enough, only changed it because I've seen rubocop complaining about it elsewhere lol :)